ansible/roles/keycloak-provision/tasks/main.yml

```diff
@@ -27,3 +27,14 @@
   with_items:
     - zip
     - unzip
+
+- name: Install pip2
+  apt:
+    name: python-pip
```

Comment:
This package will not be available in ubuntu 18 or 20 and above. We should add the ppa as a pre-requisite step before executing this.